src/main/java/pl/cleankod/exchange/core/config/ExchangeConfig.java

```diff
@@ -0,0 +1,43 @@
+package pl.cleankod.exchange.core.config;
+
+import org.springframework.boot.SpringBootConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.core.env.Environment;
+import pl.cleankod.exchange.core.gateway.AccountRepository;
+import pl.cleankod.exchange.core.gateway.AccountService;
+import pl.cleankod.exchange.core.gateway.CurrencyConversionService;
+import pl.cleankod.exchange.core.usecase.AccountServiceImpl;
+import pl.cleankod.exchange.entrypoint.AccountController;
+import pl.cleankod.exchange.entrypoint.ExceptionHandlerAdvice;
+import pl.cleankod.exchange.entrypoint.model.ValidationException;
+import pl.cleankod.exchange.provider.AccountInMemoryRepository;
+
+import java.util.Currency;
+
+@SpringBootConfiguration
+public class ExchangeConfig {
+
+    @Bean
+    AccountController accountController(AccountService accountService) {
+        return new AccountController(accountService);
+    }
+
+    @Bean
+    AccountService accountService(AccountRepository accountRepository,
+                                  CurrencyConversionService currencyConversionService,
+                                  Environment environment) {
```

Comment:
Why are you using the `Environment` instead of using a class annotated with `@ConfigurationProperties` or an `@Value` annotation?